xep-0369.xml

```diff
@@ -668,7 +679,7 @@
   <ol>
     <li>IQ.  All of the IQ errors of &rfc6120; MUST be supported.</li>
     <li>Messages.  If a message is received and an error situation is encountered, a message of type error MUST be sent back to the message sender. This message format is specified in &rfc6121; containing an error defined in &rfc6120;, which is the same error set as for IQs.</li>
-    <li>Presence.  Any responses to presence messages MUST follow the rules of &rfc6121;.</li>
+    <li>Presence. If MIX-PRESENCE is supported,  any responses to presence messages MUST follow the rules of &rfc6121;.</li>
```

Comment:
There's no such thing as a "presence message".
It's not clear to me what following the rules of 6121 would mean in this case?

Reply:
&lt;Steve&gt; Corrected

Reply:
> I’m not sure what to write here.

What is the text trying to say?